internal/fsutil/create_test.go

```diff
@@ -108,8 +169,8 @@ func (s *S) TestCreate(c *C) {
 		}
 		c.Logf("Options: %v", test.options)
 		dir := c.MkDir()
-		if test.hackdir != nil {
-			test.hackdir(c, dir)
+		if test.hackopt != nil {
+			test.hackopt(c, dir, &test.options)
 		}
```

Comment:
The code for `hackopt` should go just before calling the function that we are testing, no changes to options should happen after this call. How else can we change the options if the code will mutate them later.

Reply:
This is not still not true, `options.Path` is set two lines afterwards.

Reply:
Fixed in [4090b4c](https://github.com/letFunny/chisel/pull/17/commits/4090b4c4c27eda1bd386bb1757572a4529887ad9).